docker-bake.hcl

```diff
@@ -61,3 +61,15 @@ target "cross" {
 target "dynbinary-cross" {
     inherits = ["dynbinary", "_all_platforms"]
 }
+
+target "lint" {
+    dockerfile = "./dockerfiles/Dockerfile.lint"
+    target = "lint"
+    output = ["type=cacheonly"]
+}
+
+target "shellcheck" {
+    dockerfile = "./dockerfiles/Dockerfile.shellcheck"
+    target = "shellcheck"
+    output = ["type=cacheonly"]
+}
```

Comment:
> wdyt? Perhaps someone has `shellcheck` installed locally, and would allow them to run the script directly.

I think it's better to use a vetted/sandboxed version of shellcheck that is available from sources to avoid any drawbacks. Pretty much like golangci-lint.